src/pages/ReservationPage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Reservation from '../components/Reservation';
+
+function ReservationPage() {
+  return (
+    <Reservation />
+  );
+}
```

Comment:
- [ ]  Please, kindly refactor this function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const ReservationPage= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

That way, your app will comply with latest standards 💪

Reply:
Awesome 🚀 , we will make these changes and make a second review